src/rust/include/rust/orchard/wallet.h

```diff
@@ -263,6 +263,59 @@ void orchard_wallet_get_filtered_notes(
         push_note_callback_t push_cb
         );
 
+/**
+ * A C struct used to transfer Orchard action spend information across the FFI boundary.
+ * This must have the same in-memory representation as the `FFIActionSpend` type in
+ * orchard_ffi/wallet.rs.
+ */
+struct RawOrchardActionSpend {
+    uint32_t spendActionIdx;
+    unsigned char outpointTxId[32];
+    uint32_t outpointActionIdx;
+    OrchardRawAddressPtr* receivedAt;
+    CAmount noteValue;
+};
+
+/**
+ * A C struct used to transfer Orchard action output information across the FFI boundary.
+ * This must have the same in-memory representation as the `FFIActionOutput` type in
+ * orchard_ffi/wallet.rs.
+ */
+struct RawOrchardActionOutput {
+    uint32_t outputActionIdx;
+    OrchardRawAddressPtr* addr;
```

Comment:
Can this be referred to as "recipient" to be consistent with `FFIActionOutput`?

Reply:
Fixed in #5715